tests/model.test.js

```diff
@@ -0,0 +1,21 @@
+import MODEL_TREE from "@exabyte-io/mode.js/model_tree";
+import { expect } from "chai";
+import t from "t";
+
+import { getModelTreeByApplication } from "../src/js/model_features";
+
+describe("Model features", () => {
+    it("can be filtered by application", () => {
+        const filtered = getModelTreeByApplication({
+            tree: MODEL_TREE,
+            appName: "espresso",
+            version: "5.4.0",
+            executable: "pw.x",
+            build: "default",
+        });
+        const pbe = t.find(filtered, (node) => node.path === "/dft/gga/pbe");
```

Comment:
It seems that we are using `t` to manipulate the tree - is it possible to "outsource" the manipulation logic in `utils`, etc. to this library as well?

Reply:
I think we could outsource it, but none of the npm packages I looked through seem to fulfill the criteria we discussed. Perhaps we could add such functionality to code.js instead?